clang-tools-extra/test/clang-doc/clang-doc-project1.cpp

```diff
@@ -0,0 +1,361 @@
+// RUN: rm -rf %t && mkdir -p %t/build %t/include %t/src %t/docs
```

Comment:
I think this is a `.cp` file because it used to hold `main()`, right? Probably just `basic-project.test` is fine. or `shape-project.test`. It's typical to use a `.test` extension in files that are only used to hold the `RUN`/`CHECK` lines. `lld/test/ELF` has a lot of examples like that. I'm sure there are others too.